iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -153,18 +172,21 @@ inline void storable_function<Capacity, signature<ReturnType, Args...>>::storeFu
     auto ptr = m_storage.template allocate<StoredType>();
     cxx::Expects(ptr != nullptr);
 
+    // AXIVION Next Construct AutosarC++19_03-A18.5.10: false positive, m_storage.allocate<T>
```

Comment:
Pointer trickery with invariants not known to Axivion. Requires semantic understanding that allocate provides an aligned pointer for `StoredType` which is non-trivial to infer automatically.